fixtures/s3/docker-compose-ceph-rados.yml

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+version: '3.8'
+
+services:
+  ceph-rgw:
+    image: ceph/daemon:latest
+    network_mode: host
+    environment:
+      - CEPH_DAEMON=rgw
+      - RGW_FRONTEND_PORT=8000
+      - RGW_CIVETWEB_PORT=8000
+      - CEPH_PUBLIC_NETWORK=0.0.0.0/0
+      - CEPH_CLUSTER_NAME=myceph
+      - CEPH_RGW_FRONTEND_TYPE=beast
+    volumes:
+      - /var/lib/ceph/:/var/lib/ceph/
+
+  rgw-admin-user-setup:
```

Comment:
Running setup in `docker-compose` seems not a good idea. We can run it after `ceph-rgw` is running.